### PR TITLE
Fixing quality flaws

### PR DESCRIPTION
**Pull request by tvsonar**

### CODE REVIEW

**Comment by tvsonar** [comment]
Enable all rules in the ruleset file.

File: src/SonarLint/Common/FixAllProviders/DocumentBasedFixAllProvider.cs [hunk 1]
```diff
@@ -55,8 +55,7 @@ private DocumentBasedFixAllProvider()
         private static string GetFixAllTitle(FixAllContext fixAllContext)
         {
             var diagnosticIds = fixAllContext.DiagnosticIds;
-            string diagnosticId;
-            diagnosticId = diagnosticIds.Count() == 1
+            var diagnosticId = diagnosticIds.Count() == 1
```
Comment (dbolkensteyn) [inline src/SonarLint/Common/FixAllProviders/DocumentBasedFixAllProvider.cs]: @tvsonar is this `optimization` even required?

**Comment by dbolkensteyn** [comment]
LGTM

**Comment by tvsonar** [comment]
This should also include the changes proposed in https://github.com/SonarSource/sonarlint-vs/pull/566#discussion_r44389105

**Pull request merged**